src/pages/index.js

```diff
@@ -1,150 +1,76 @@
-import * as React from "react"
+import * as React from "react";
+import { Swipeable, direction } from "react-deck-swiper";
+import getData from "../data";
 
-// styles
-const pageStyles = {
-  color: "#232129",
-  padding: "96px",
+const container = {
```

Comment:
May we use SCSS modules?

Reply:
Of course, this is just a dummy rendering. This PR is about the swipe effect and the data management.